<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisitBeforeRecordClient</class>
 <widget class="QDialog" name="VisitBeforeRecordClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Протокол обращений пациента по предварительной записи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>с</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBeginDateVisitBeforeRecordClient">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>по</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDateVisitBeforeRecordClient">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="COrgStructureComboBox" name="cmbOrgStructureVisitBeforeRecordClient"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbSpecialityVisitBeforeRecordClient"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPersonVisitBeforeRecordClient"/>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QCheckBox" name="chkNoVisitBeforeRecordClient">
     <property name="text">
      <string>Показывать записи без обращения</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTextBrowser" name="txtVisitBeforeRecordClient"/>
     <widget class="CTableView" name="tblVisitByQueue"/>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>152</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="btnCreateVisitBeforeRecordClient">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Начать выполнение</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="btnPrintVisitBeforeRecordClient">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Печать</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="btnCancelVisitBeforeRecordClient">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBeginDateVisitBeforeRecordClient</tabstop>
  <tabstop>edtEndDateVisitBeforeRecordClient</tabstop>
  <tabstop>cmbOrgStructureVisitBeforeRecordClient</tabstop>
  <tabstop>cmbSpecialityVisitBeforeRecordClient</tabstop>
  <tabstop>cmbPersonVisitBeforeRecordClient</tabstop>
  <tabstop>chkNoVisitBeforeRecordClient</tabstop>
  <tabstop>txtVisitBeforeRecordClient</tabstop>
  <tabstop>tblVisitByQueue</tabstop>
  <tabstop>btnCreateVisitBeforeRecordClient</tabstop>
  <tabstop>btnPrintVisitBeforeRecordClient</tabstop>
  <tabstop>btnCancelVisitBeforeRecordClient</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
